user_guide/common_features.rst

```diff
@@ -49,10 +49,32 @@ prior agreements.
 
 In OroCRM, all of this data is tracked using three types of records: channels, customers, and accounts.
 
+
+Understanding Accounts, Channels and Customers
```

Comment:
Comment: In my edits, I'm using the serial comma. Feel free to ignore if not inline with your internal style guide.